specifyweb/specify/management/commands/base_specify_migration.py

```
@@ -10,43 +11,68 @@
-                cursor.execute("""
-                    INSERT INTO django_migrations (app, name, applied)
-                    VALUES ('specify', '0001_initial', NOW());
-                """)
+        use_override = bool(options.get('use_override', False))
+        alias = options["database"]
+        conn = connections[alias]
+        logger.info(f"Running base_specify_migration using database alias '{alias}'")
+
+        try:
+            transaction.atomic(using=alias)
+        except:
```

Ruff (0.15.14)

[error] Do not use bare `except`

(E722)